src/wallet/rpcwallet.cpp

```diff
@@ -4420,6 +4454,9 @@ size_t EstimateTxSize(
     }
     txsize += CTXOUT_REGULAR_SIZE * taddrRecipientCount;
 
+    if (orchardRecipientCount) {
```

Comment:
nit:
```suggestion
    if (orchardRecipientCount > 0) {
```
(this change is not needed because this variable is unsigned, and since its name includes `Count`, it's pretty clear that it's not a boolean, but the comparison just makes it absolutely clear to the reader)

Reply:
Force-pushed to address review comments